src/Persistence/Templates/BuiltInTemplateIds.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.PowerPlatform.PowerApps.Persistence.Templates;
+
+public sealed class BuiltInTemplateIds
```

Comment:
`class` for holding `const`s and/or `static`s shouldn't be instantiable: 
```suggestion
public static class BuiltInTemplateIds
```

Reply:
good point